src/components/screens/timebank-screen.tsx

```diff
@@ -30,28 +30,45 @@ const TimebankScreen = () => {
   const [personTotalTime, setPersonTotalTime] = useAtom(personTotalTimeAtom);
   const [personDailyEntry, setPersonDailyEntry] = useAtom(personDailyEntryAtom);
   const [dailyEntries, setDailyEntries] = useAtom(dailyEntriesAtom);
+  const [selectedEmployee] = useState<number | null>(
```

Comment:
I may not be necessary to define types here, they can be inferred. Also do you need useState for this? It could also be just `const selectedEmployee = ...`

Reply:
Removed useState.